src/Components/Register/Register.jsx

```diff
@@ -19,42 +19,47 @@ export default class Register extends Component {
 
   handleSubmit(event) {
     event.preventDefault();
+    const { dispatch, firstName, lastName, email, password } = this.props;
 
-    if (this.props.confirmPassword === this.props.password) {
-      const { dispatch, firstName, lastName, email, password } = this.props;
-      dispatch(register({ firstName, lastName, email, password }, dispatch ))
+    if (!this.props.email.includes('.', this.props.email.indexOf('@') + 2) || this.props.email.substring(this.props.email.lastIndexOf('.')).length <= 2) {
```

Comment:
Since we're destructuring `email` on line 22, you don't need to include `this.props` on this line

Reply:
Resolved.